### DISCUSSION TITLE
IncludeDetails vs "IncludeFields"?

### DISCUSSION THREAD

**Discussion by TonyValenti** [abpframework/abp#7949] [Q&A]
I used to use a system named Clio which allows you to explicitly state what fields you want back from the database.
https://app.clio.com/api/v4/documentation#section/Fields

For example, a url like this:
https://app.clio.com/api/v4/matters?fields=id,etag,description,client{name}

Would return a value like:
```
{
  "id":1,
  "etag":"...",
  "description": "some value",
  "client": {
     "name": "the client"
  }
}
```

Is there a way to enable that level of efficiency with ABP?

**Comment by maliming** [comment]
What is your Database Provider? EF Core or MongoDB?

**Reply by TonyValenti** [reply]
EFCore.

**Reply by maliming** [reply]
hi

Maybe you can research https://dynamic-linq.net/ this not related to abp.
